src/manage_sql_nvts.c

```diff
@@ -1410,6 +1410,75 @@ check_preference_names (const gchar *table)
        table);
 }
 
+/**
+ * @brief Update config preferences where the name has changed in the NVTs.
+ *
+ * @param[in]  trash              Whether to update the trash table.
+ * @param[in]  modification_time  Time NVTs considered must be modified after.
+ */
+static void
+check_preference_names (int trash, time_t modification_time)
+{
+  iterator_t prefs;
+
+  sql_begin_immediate ();
+
+  init_iterator (&prefs,
+                 "WITH new_pref_matches AS"
+                 " (SELECT substring(nvt_preferences.name,"
+                 "                   '^([^:]*:[^:]*)') || ':%%' AS match,"
+                 "          name AS new_name"
+                 "     FROM nvt_preferences"
+                 "    WHERE substr (name, 0, position(':' IN name))"
```

Comment:
```suggestion
                 "    WHERE substr (name, 0, position (':' IN name))"
```

At least we should be consistent about this within a single statement :)